proto/table_marshal.go

```diff
@@ -213,7 +227,15 @@ func (u *marshalInfo) size(ptr pointer) int {
 // cachedsize gets the size from cache. If there is no cache (i.e. message is not generated),
 // fall back to compute the size.
 func (u *marshalInfo) cachedsize(ptr pointer) int {
-	if u.sizecache.IsValid() {
+	if u.hassizer {
```

Comment:
These will now never be cached, I can see that it will work, but a cachedsize, should win in speed, I would think.

Reply:
yeh. I agree.